apps/web/src/lang/fr.json

```diff
@@ -6158,6 +6166,10 @@
     "defaultMessage": "« Je suis membre d’une Première Nation »",
     "description": "Text for the option to self-declare as first nations"
   },
+  "WppiY9": {
+    "defaultMessage": "Afin d’écourter le processus de demande, l’information n’est recueillie que sur certaines compétences particulières à l’étape de la demande. Celles-ci sont identifiées individuellement sous le nom de la compétence en se servant de la mention \"Évaluées à l’étape initiale de la demande\". D’autres évaluations suivront plus tard au cours de votre processus de demande si elle s’avère réussie.",
```

Comment:
Are we using double quotes in French for the purpose of highlighting copy?

Reply:
Fixed in [remove double quotes in string](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9145/commits/626cbe1a124d15733ac60ab8d4b8aa7f1161941c)